NCU-Project/Dogs/DogsTableViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DogsTableViewController.swift
+//  NCU-Project
+//
+//  Created by Simon Rowlands on 15/10/2018.
+//  Copyright © 2018 simonrowlands. All rights reserved.
+//
+
+import UIKit
+
+import RxSwift
+import RxCocoa
+
+class DogsTableViewController: UITableViewController {
```

Comment:
we can use `final` here

Reply:
oh didn't notice this before, I wouldn't recomment using `UITableViewController`. its more flexible to just create `UITableView`in case we want to add custom functioanlity into it